docs/README.md

```diff
@@ -1,20 +1,24 @@
 # Overview
 ![k0s logo](img/k0s-logo-full-color.svg)
 
-k0s is an all-inclusive Kubernetes distribution with all the required bells and whistles preconfigured to make building a Kubernetes clusters a matter of just copying an executable to every host and running it.
+k0s is an all-inclusive Kubernetes distribution, configured with all of the features needed to build a Kubernetes cluster simply by copying and running an executable file on each target host.
 
 ## Key Features
-- Packaged as a single static binary
-- Self-hosted, isolated control plane
-- Variety of storage backends: etcd, SQLite, MySQL (or any compatible), PostgreSQL
-- Elastic control-plane
+- Available as a single static binary
+- Offers a self-hosted, isolated control plane
+- Supports a variety of storage backends, including etcd, SQLite, MySQL (or any
```

Comment:
Remove the manual line changes (new line characters) from the middle of the sentence (this is general comment for all changes). Otherwise it becomes difficult to modify the text later on and you’d need to change all the line changes to different place. Line changes can be left to the text editor and browser, which will automatically scale the text according to the screen size. The whole docs has been written without manual line changes.